src/person/db.go

```diff
@@ -23,19 +22,18 @@ func init() {
 		os.Getenv("MYSQLPORT"),
 		os.Getenv("MYSQLDB"),
 	)
-	// Attempt to establish a new SQL driver connection (does not actually connect to the SQL server)
-	database, err := sql.Open("mysql", d)
-	// Something went wrong when trying to establish a driver connection to SQL
+
+	var err error
+	db, err = sql.Open("mysql", d)
```

Comment:
You can assign right here instead of below just as long as you use the `=` operator instead of the `:=` operator, since the latter would shadow `db`.